plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -254,7 +254,9 @@ private function init_hooks() {
 		add_action( 'init', array( 'WC_Emails', 'init_transactional_emails' ) );
 		add_action( 'init', array( $this, 'add_image_sizes' ) );
 		add_action( 'init', array( $this, 'load_rest_api' ) );
-		add_action( 'init', array( 'WC_Site_Tracking', 'init' ) );
+		if ( $this->is_request( 'admin' ) || ( $this->is_rest_api_request() && ! $this->is_store_api_request() ) || ( defined( 'WP_CLI' ) && WP_CLI ) ) {
```

Comment:
in the similar way with other properties, maybe you could create a `$this->is_wp_cli`. Just a minor suggestion.

Reply:
Thought about it, but the CLI isn't WC specific so it felled a little strange to add a `is_wp_cli` to the WooCommerce class